playbooks/demo_vios_update_and_upgrade.yml

```diff
@@ -0,0 +1,55 @@
+- name: HMC VIOS update and upgrade
```

Comment:
demonstration of upgrading the VIOS version to 4.1.0.10 and then updating the minor patch version 4.1.0.21